weed/storage/volume.go

```diff
@@ -133,6 +133,31 @@ func (v *Volume) ContentSize() uint64 {
 	return v.nm.ContentSize()
 }
 
+func (v *Volume) IsEmpty() (bool, error) {
+	v.dataFileAccessLock.RLock()
+	defer v.dataFileAccessLock.RUnlock()
+	return v.doIsEmpty()
+}
+
+func (v *Volume) doIsEmpty() (bool, error) {
+	if v.DataBackend != nil {
+		datFileSize, _, e := v.DataBackend.GetStat()
+		if e != nil {
+			glog.V(0).Infof("Failed to read file size %s %v", v.DataBackend.Name(), e)
+			return false, e
+		}
+		if datFileSize > 8 {
```

Comment:
change `8` to `NeedlePaddingSize`([/weed/storage/types/needle_types.go#L39](https://github.com/seaweedfs/seaweedfs/blob/7e0fc5e2fb808ffc2447353cc48cdfb24f0c16d3/weed/storage/types/needle_types.go#L39))?